### ISSUE TITLE
Name inconsistency between glyma gnm2 and gnm4 in GFFs

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/datastore-issues#7]
I use the "Name" attribute for secondaryIdentifier, which I merge on, so this matters in the mine loads.

Glycine_max/Wm82.gnm2.ann1.RVB6/glyma.Wm82.gnm2.ann1.RVB6.gene_models_main.gff3
```
glyma.Wm82.gnm2.Gm01    phytozomev10    gene    27355   28320   .       -       .       ID=glyma.Wm82.gnm2.ann1.Glyma.01G000100;Name=glyma.Glyma.01G000100;ancestorIdentifier=Glyma01g00210.v1.1;Note=protein PHYLLO%2C chloroplastic-like isoform X1 [Glycine max];Dbxref=Gene3D:G3DSA:3.30.390.10;
```

Glycine_max/Wm82.gnm4.ann1.T8TQ/glyma.Wm82.gnm4.ann1.T8TQ.gene_models_main.gff3:
```
glyma.Wm82.gnm4.Gm01    phytozomev13    gene    27344   28430   .       -       .       ID=glyma.Wm82.gnm4.ann1.Glyma.01G000100;Name=Glyma.01G000100;ancestorIdentifier=Glyma.01G000100.Wm82.a2.v1;Dbxref=Gene3D:G3DSA:3.30.390.10;Note=protein PHYLLO%2C chloroplastic-like isoform X1 [Glycine max]
```

**Comment by sammyjava** [comment]
(Not the end of the world since I can pull it from the full-yuck ID but we should have things be consistent.)

**Comment by cann0010** [comment]
I see that gnm2.ann1 has Name=glyma.Glyma.01G000100   
 and that  gnm4.ann1 has Name=Glyma.01G000100

In the "display name" in the browsers at SoyBase, we are using the latter form (Glyma.01G000100) for both assemblies - but this is just for aesthetics  within the brewers.
@adf-ncgr - do you recall if we have a standard form for the Name attribute?

**Comment by sammyjava** [comment]
I use Glyma.01G000100 as well as secondary identifier (and what appears in bold on the gene reports).

**Comment by adf-ncgr** [comment]
traditionally, we prefix the Name with the gensp for use of a slightly-less-yucky-but-still-informative label in multispecies contexts (like trees and GCV). But we also have found ourselves sliding to more-but-not-quite-fully-yucky in cases like chickpea when we have multiple genome sequences within the species. There may be reasons to say in the GFF we should just have ID=Name and let downstream consumers decide how much yuck they can drop for their labelling purposes.

**Comment by sammyjava** [comment]
I can do that. But I think that soybean gnm2 Glyma.01G000100 should have the same name as soybean gnm4 Glyma.01G000100. Since they're the same gene. The Name attribute is non-unique and is meant to provide some user-friendly (consistent, familiar, informative) naming. But I'm happy to implement that off of the ID=primaryIdentifier.

**Comment by adf-ncgr** [comment]
will be fixed as part of #16
